src/lucky_record/associations.cr

```diff
@@ -2,13 +2,35 @@ module LuckyRecord::Associations
   macro has_many(type_declaration)
     {% assoc_name = type_declaration.var }
     {% model = type_declaration.type %}
+    {% foreign_key = "#{@type.name.underscore}_id".id %}
+    @_preloaded_{{ assoc_name }} : Array({{ model }})?
+    setter _preloaded_{{ assoc_name }}
+
+    class BaseQuery < LuckyRecord::Query
+      def preload_{{ assoc_name }}(preload_query : {{ model }}::BaseQuery = {{ model}}::BaseQuery.new)
```

Comment:
The second `{{ model}}` is missing a space before the curly brace.

Reply:
😱